lib/micro-enum.js

```diff
@@ -1 +1 @@
-Enum=function(){let v=arguments,s={all:[],keys:v};for(let i=v.length;i--;)s[v[i]]=s.all[i]=i;return s;}
+module.exports=Enum=((...l)=>(s={all:[],keys:l},l.map((l,a)=>s[l]=s.all[a]=a),s));
```

Comment:
This is awesome!

Can you help me understand why you reassign `module.exports` here as well as in `index.js`?